src/draw.c

```diff
@@ -303,17 +303,17 @@ extern char* get_spec(int i)
 		case  2: return "z"; break;
 		case  3: return "coil"; break;
 		case  4: return "map"; break;
-		case  5: return "n"; break;
-		case  6: return "o"; break;
-		case  7: return "p"; break;
-		case  8: return "q"; break;
-		case  9: return "slice"; break;
+		case  5: return "te"; break;
+		case  6: return "coeff"; break;
+		case  7: return "coeff2"; break;
+		case  8: return "iter"; break;
+		case  9: return "cshift"; break;
```

Comment:
I am not so much a fan of this naming concention. Also some stuff is outdated (cshift).